FV3/conf/detect_mpi_links_args.sh

```diff
@@ -2,11 +2,17 @@
 
 function mpichLDFlags {
     # -show prints includes as well as the exe...remove these
-    mpifort -show | sed 's/\(-I[^ ]*\|^[^ ]*\)//g'
+    if command -v mpifort &> /dev/null
```

Comment:
On Gaea a different compiler is used so without these guards this script fails.  It's not clear what the replacement should be, and empirically it seems we can get away without defining this variable there, so for now I just don't do anything if these commands cannot be found.